pkg/libvirttools/libvirt_domain.go

```diff
@@ -30,6 +31,18 @@ type libvirtDomainConnection struct {
 	conn libvirtConnection
 }
 
+// TODO: runtime issue: https://github.com/futurewei-cloud/arktos-vm-runtime/issues/50
+//       multiple sizes of devices, and, numa node setting
+// default mem chip size set to 128 MiB
```

Comment:
Can we change it to a larger size in this PR? With 2G as the max increase this feature's value is very limited.  

After this PR we can quickly do some per-VM size changes. For example, set the mem chip size to the round-up of a VM's initial memory request.  So a VM's memory can at most increase to its 16 times during run time. That probably provide more value in heavy workload scenarios.

Reply:
Have tested with 512Mi as the default mem device size and it works fine. so made the change.
Noticed that as you mentioned, we have a task for multiple size of mem devices support with hotplug/unplug feature. which will allow up to 16 * GB level changes.